### PR TITLE
flag msg in shop (same way as for bank)

### PR DESCRIPTION
**Pull request by fokkonaut**

### CODE REVIEW

File: src/game/server/entities/character.cpp [hunk 1]
```diff
@@ -3767,7 +3767,21 @@ void CCharacter::HandleTiles(int Index)
...
 
 		if (Server()->Tick() % 50 == 0)
-			GameServer()->SendBroadcast("~ S H O P ~", m_pPlayer->GetCID(), 0);
+		{
+			if (((CGameControllerDDRace*)GameServer()->m_pController)->HasFlag(this) != -1) //has flag
+			{
+				if (m_pPlayer->m_AccountID <= 0) //only print stuff if player is not logged in while flag carry
+				{
+					GameServer()->SendBroadcast("~ S H O P ~", m_pPlayer->GetCID(), 0);
+					dbg_msg("debug", "shoptile");
```
Comment (ChillerDragon) [inline src/game/server/entities/character.cpp]: remove debug messages pls

File: src/game/server/entities/character.cpp [hunk 2]
```diff
@@ -3767,7 +3767,21 @@ void CCharacter::HandleTiles(int Index)
...
+				}
+			}
+			else // no flag --> print always
+			{
+				GameServer()->SendBroadcast("~ S H O P ~", m_pPlayer->GetCID(), 0);
+				dbg_msg("debug", "shoptile");
```
Comment (ChillerDragon) [inline src/game/server/entities/character.cpp]: here aswell

**Review by ChillerDragon — CHANGES_REQUESTED** [review]

File: src/game/server/entities/character.cpp (hunk 2 above)
Comment (fokkonaut) [reply src/game/server/entities/character.cpp]: no u made it too

**Comment by fokkonaut** [comment]
merge

**Review by ChillerDragon — APPROVED** [review]

**Pull request merged**